.old_travis.yml

```diff
@@ -0,0 +1,45 @@
+language: java
```

Comment:
I do not understand the purpose of this file. If it's just an old copy of `.travis.yml` it should not be in the repo: we use Git for a reason. If it's something else (or it has a reason to exist) that should be documented. Right now I have no idea if it's here by accident or not and under what circumstances it should be modified or can be removed from the repo.